tests/NetEscapades.EnumGenerators.Tests/TestHelpers.cs

```diff
@@ -344,4 +345,37 @@ public Options(LanguageVersion LanguageVersion,
         public string[]? Stages { get; }
 
     }
+
+    public static async Task<ImmutableArray<Diagnostic>> GetAnalyzerDiagnosticsAsync(
```

Comment:
Don't add this new method, update the existing methods to include the new analyzer in all the compilation tests